packages/backend/src/database/migrations/20230329082135_scheduler_log_optional_values.ts

```diff
@@ -0,0 +1,20 @@
+import { Knex } from 'knex';
+
+const SchedulerLogTableName = 'scheduler_log';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable(SchedulerLogTableName, (table) => {
+        table.uuid('scheduler_uuid').nullable().alter();
+        table.string('job_group').nullable().alter();
+    });
+}
+
+export async function down(knex: Knex): Promise<void> {
+    // Delete first all nullable values
+    await knex(SchedulerLogTableName).delete().whereNull('scheduler_uuid');
+
+    await knex.schema.alterTable(SchedulerLogTableName, (table) => {
```

Comment:
This rollback will not work if there are null values in the table. You'd need to delete rows first before doing this!

Reply:
that's exactly what's  happening 2 lines above, right ? :thinking:

Reply:
🤦

Reply:
I'm so sorry, I totally missed that!